src/docs/get-started/codelab-web.md

```diff
@@ -1028,6 +1033,7 @@ Dart DevTools, or Flutter animations, see the following:
 [DevTools documentation]: /docs/development/tools/devtools
 [DevTools installed]: /docs/development/tools/devtools/overview#how-do-i-install-devtools
 [DartPad troubleshooting page]: {{site.dart-site}}/tools/dartpad/troubleshoot
+[didUpdateWidget][]: https://api.flutter.dev/flutter/widgets/State/didUpdateWidget.html
```

Comment:
Delete the empty brackets. Also, please use the API macro:   `{{site.api}}/flutter/widgets/...`